main.tf

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2020 Quortex
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+# There are 3 types of load balancers in AWS: 
+# - Classic
+# - NLB (Network Load Balancer)
+# - ALB (Application Load Balancer)
+#
+# Here an Application Load Balancer is created.
+# It listens for HTTP and HTTPS, and forwards HTTPS to instances of the 
+# target group.
+# The target group is made of instances, which are the instances of the
+# autoscaling group. 
```

Comment:
This block of comment is unnecessary here

Reply:
Fixed